include/mbgl/style/light.hpp

```diff
@@ -1,52 +1,48 @@
+// This file is generated. Do not edit.
+
 #pragma once
 
-#include <mbgl/style/property_value.hpp>
-#include <mbgl/style/transition_options.hpp>
-#include <mbgl/style/types.hpp>
-#include <mbgl/style/position.hpp>
-#include <mbgl/util/color.hpp>
-#include <mbgl/util/indexed_tuple.hpp>
+#include <mbgl/style/light_properties.hpp>
```

Comment:
This isn't used in the header, right? It can move to the `.cpp` file an the `src` directory tree.